packages/mrm-task-eslint/index.js

```diff
@@ -67,15 +72,24 @@ module.exports = function task({
 
 	// .eslintrc.json
 	const eslintrc = json(configFile, legacyConfig);
-	const hasCustomPreset = _.castArray(eslintrc.get('extends', [])).find(x =>
-		x.startsWith(eslintPreset)
-	);
+	const extnds = _.castArray(eslintrc.get('extends', []));
+	const hasCustomPreset =
+		extnds.length &&
```

Comment:
No need for the lengths check here:

```suggestion
```

Reply:
We actually do need it here because with an empty array, `[].every(...)` will give `true` but we don't want it to be `true` if it is empty.

Reply:
Then, why .every() and not .some() here? We want to know that there's a custom preset, not that all of them are custom? What am I missing?

Reply:
If `hasCustomPreset` ends up true (because there is a preexisting `extends` but every preset in the `eslintPreset` option array can be found in the config) then we skip modifying `extends`. 

This kind of parallels the old behavior but admittedly it is probably more convoluted than it needs to be. I guess we could always attempt to set or merge `extends` unless the `eslintPreset` option can somehow end up empty (without applying the default).

Reply:
It also might not be fully safe to use `startsWith` here either since there could be a config like `eslint-config-airbnb` and a config `eslint-config-airbnb-special`.